net/smartdns/Makefile

```diff
@@ -1,18 +1,18 @@
 #
-# Copyright (c) 2018-2023 Nick Peng (pymumu@gmail.com)
+# Copyright (c) 2018-2024 Nick Peng (pymumu@gmail.com)
 # This is free software, licensed under the GNU General Public License v3.
 #
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=smartdns
-PKG_VERSION:=1.2023.43
+PKG_VERSION:=1.2024.45
```

Comment:
This version does not exists by looking at https://github.com/pymumu/smartdns/releases/tag/Release45

Reply:
I used 1.YYYY.VV because that's what previous versions use.

Their own releases are built with a [makeshift script](https://github.com/pymumu/smartdns/blob/Release45/package/openwrt/make.sh) that uses a different version scheme, and the releases themselves uses yet another version scheme with the script's version scheme retained.

Reply:
Oh my god... this is so wrong. Just report it to them. What the hell. Why would you do such thing? It does not make any sense.